packages/form-plugin/src/directive.ts

```diff
@@ -25,15 +36,22 @@ export class FormDirective implements OnInit, OnDestroy {
   @Input('ngxsFormClearOnDestroy')
   clearDestroy = false;
 
+  /**
+   * `valueChangesStrategy` is noop by default but can be overwritten via binding
+   */
+  @Input('ngxsFormValueChangesStrategy')
+  valueChangesStrategy: Type<
+    NgxsFormPluginValueChangesStrategy
+  > = NoopNgxsFormPluginValueChangesStrategy;
```

Comment:
And if I get tired every time for each form to redefine the strategy?

Reply:
Your suggestions?

Reply:
support 

```ts
NgxsFormPlugin.forRoot({ defaultStategy: NoopNgxsFormPluginValueChangesStrategy })
```

Reply:
What is this for? Noop strategy is used by default.

Reply:
But if I want to make a different strategy in the application for all forms at once

```ts
NgxsFormPlugin.forRoot({ defaultStategy: DeepEqualNgxsFormPluginValueChangesStrategy })
```

or 

```ts
NgxsFormPlugin.forRoot({ defaultStategy: CustomStrategy })
```

Reply:
That makes sense.

Tho to avoid making changes and then revert it back, let's wait for @markwhitfeld response. If it's OK then I will add this option.